backup-stores/gcs/src/main/java/io/camunda/zeebe/backup/gcs/ManifestManager.java

```diff
@@ -137,5 +154,41 @@ public void markAsFailed(final BackupIdentifier id, final String failureReason)
     }
   }
 
+  public Collection<Manifest> listManifests(final BackupIdentifierWildcard wildcard) {
+    final var spliterator =
+        Spliterators.spliteratorUnknownSize(
+            client
+                .list(bucketInfo.getName(), BlobListOption.prefix(prefix))
```

Comment:
:wrench: @oleschoenburg mentions that we might can improve here the filtering, and build the prefix/path based on the given wildcard.